### PR TITLE
README updates to setup connect to sdr-infra.stanford.edu

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Updates that go along with https://github.com/sul-dlss/DeveloperPlaybook/pull/229

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,68 @@ Make sure that:
...
+    GSSAPIAuthentication yes  
+    GSSAPIDelegateCredentials yes  
+    # Specify your Stanford username here  
+    User YOURSUNETIDHERE
+```
+
+2.) Add your github key to your local ssh agent
+
+```
+ssh-add ~/.ssh/id_ed25519`
```
Comment (mjgiarlo) [inline README.md:51]: Add comment?

`# Or whatever the path is to the private key you've added to GitHub`

**Review by mjgiarlo — APPROVED** [review]

**Review by mjgiarlo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -12,13 +12,81 @@ Make sure that:
...
+Passcode or option (1-3):
+```
+
+Once connected, you can proceed.
+
+### Configure bundler for your local path
+
+Set the bundler path:
+```
+$ bundle config --global path /home/[username]/.vendor/bundle
```
Comment (lwrubel) [inline README.md:82]: I like to use the copy button from Github READMEs, so it's nice to not have the `$` in commands.

File: README.md [hunk 3]
```diff
@@ -12,13 +12,81 @@ Make sure that:
...
+```
+
+Setup contribsys gem authentication (Sidekiq pro):
+```
+$ bundle config gems.contribsys.com USER:PASS
+```
```
Comment (lwrubel) [inline README.md:88]: Where do I find the values for this?

Comment (lwrubel) [reply README.md:88]: Oops, never mind, I see this is covered above.

File: README.md (hunk 2 above)
Comment (aaron-collier) [reply README.md]: Thanks @lwrubel , removed :-)

**Pull request merged**